internal/controllers/manager.go

```diff
@@ -0,0 +1,84 @@
+package controllers
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	"github.com/tilt-dev/tilt/internal/store"
+	corev1alpha1 "github.com/tilt-dev/tilt/pkg/apis/core/v1alpha1"
+	"github.com/tilt-dev/tilt/pkg/logger"
+	"github.com/tilt-dev/tilt/pkg/model"
+)
+
+type TiltServerControllerManager struct {
+	config *rest.Config
+	cancel context.CancelFunc
+}
+
+var _ store.SetUpper = &TiltServerControllerManager{}
+var _ store.Subscriber = &TiltServerControllerManager{}
+var _ store.TearDowner = &TiltServerControllerManager{}
+
+func ProvideRESTConfig(apiserverHost model.WebHost, apiserverPort model.WebPort) *rest.Config {
```

Comment:
This can/should be replaced by just injecting `server.APIServerConfig` into the controller manager

Reply:
Made this change in 81f95d52c1a61c5221aa4cd35a5ec6d77996a12b and got rid of `ProvideRESTConfig()`